crypto/rand/drbg_lib.c

```diff
@@ -72,6 +72,7 @@ RAND_DRBG *RAND_DRBG_new(int type, unsigned int flags, RAND_DRBG *parent)
     }
     drbg->size = RANDOMNESS_NEEDED;
     drbg->randomness = ucp;
+    drbg->fork_count = rand_fork_count;
 
```

Comment:
I just removed the line and left a single block of init statements/functions.  thanks.  assuming the +1 still holds.